Работа №1

Divium-l opened the pull request:
1. Задание 1-1 
2. Блок-схема

Divium-l committed:
Работа №1

Задание 1-1 и блок-схема

Divium-l committed:
Работа №1

Добавлен вывод исходных данных

MarinaPaley commented on task-1/task-1.cpp:
```diff
@@ -0,0 +1,17 @@
+﻿#include <iostream>
+#include <math.h>
```
не нужно использоваь библиотеку языка `C`. Используйте [C++](https://docs.microsoft.com/ru-ru/cpp/c-runtime-library/math-constants?view=msvc-160)

MarinaPaley reviewed (changes requested):
Сделайте проверкку вычислений в любом "калькуляторе"

Divium-l committed:
Работа №1 Исправление

1. Макрофункция заменена на нормальную
2. Теперь используется библиотека C++
3. Произведена проверка вычислений

MarinaPaley reviewed (approved):
Хорошо, мержуйте!
